runtime/runtime_test.go

```diff
@@ -0,0 +1,89 @@
+package runtime
+
+import (
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"reflect"
+	"testing"
+)
+
+func getRuntimeBlob() (n int64, err error) {
+	out, err := os.Create("polkadot_runtime.compact.wasm")
+	if err != nil {
+		return 0, err
+	}
+	defer out.Close()
+
+	resp, err := http.Get("https://github.com/w3f/polkadot-re-tests/blob/master/polkadot-runtime/polkadot_runtime.compact.wasm?raw=true")
+	if err != nil {
+		return 0, err
+	}
+	defer resp.Body.Close()
+
+	n, err = io.Copy(out, resp.Body)
+	return n, err
+}
+
+func TestNewVM(t *testing.T) {
+	_, err := getRuntimeBlob()
```

Comment:
Please check if it exists already or not

Reply:
I have added that to the getRuntimeBlob function so it only grabs it if it doesnt exist